src/CosmosDB/CosmosDB/help/Get-AzManagedCassandraDatacenter.md

```diff
@@ -0,0 +1,114 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.CosmosDB.dll-Help.xml
+Module Name: Az.CosmosDB
+online version: https://docs.microsoft.com/en-us/powershell/module/az.cosmosdb/get-azmanagedcassandradatacenter
+schema: 2.0.0
+---
+
+# Get-AzManagedCassandraDatacenter
+
+## SYNOPSIS
+Gets the ManagedCassandra Datacenters.
+
+## SYNTAX
+
+```
+Get-AzManagedCassandraDatacenter -ResourceGroupName <String> -ClusterName <String> [-DataCenterName <String>]
```

Comment:
Please refer to the design https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/949